src/main/java/com/uid2/shared/secure/azurecc/MaaTokenPayload.java

```diff
@@ -7,8 +7,14 @@
 @Builder(toBuilder = true)
 public class MaaTokenPayload {
     public static final String SEV_SNP_VM_TYPE = "sevsnpvm";
+    public static final String AZURE_CC_PROTOCOL = "azure-cc";
```

Comment:
i dont think we can change the  "azure-cc" name now as it's hardcoded/used in other places but the variable name - should it be something more specific than azure_CC? if not probably should add comments to say the difference between this and AKS_Protocol?